pkgs/by-name/ph/phel/package.nix

```diff
@@ -5,15 +5,15 @@
   versionCheckHook,
 }:
 
-php.buildComposerProject (finalAttrs: {
+php.buildComposerProject2 (finalAttrs: {
```

Comment:
Good catch. I thought I moved all of them.

Reply:
It's on release-24.11 :)